src/apps/jsgeneric/oe_sign.conf

```diff
@@ -1,6 +1,6 @@
 # Enclave settings:
 Debug=1
-NumHeapPages=32768
+NumHeapPages=131072
```

Comment:
Why is this required?

Reply:
The js impl seems to use a little more memory (I haven't checked how much, this is enough but probably too much).